### DISCUSSION TITLE
CUSA00557 - The Last of Us Remastered [1.11]

### DISCUSSION THREAD

**Discussion by ichopster** [GoldHEN/GoldHEN_Cheat_Repository#3] [Cheat requests]
Please add CUSA00557 - The Last of Us Remastered [1.11]

**Comment by kmeps4** [comment]
Will check if any available...

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
